vivisect/qt/funcgraph.py

```diff
@@ -243,6 +258,7 @@ def __init__(self, vw, vwqgui):
         self.addHotKeyTarget('funcgraph:inczoom', self._hotkey_inczoom)
         self.addHotKey('ctrl+-', 'funcgraph:deczoom')
         self.addHotKeyTarget('funcgraph:deczoom', self._hotkey_deczoom)
+        self.addHotKey('f5', 'funcgraph:refresh')
 
```

Comment:
because refresh is potentially a long process, i don't see any cases at the moment where the user shouldn't be involved in a refresh.  if we can shrink the overhead of redrawing, adding comments, changing va names, and probably several others should cause a redraw.